### PR TITLE
Support local force fields when planning a network

### PR DESCRIPTION
**Pull request by jthorton**
## Summary of changes
Adds an experimental version of the `plan_rbfe_network.py` script which allows for a local bespoke force field file to be used in the protocol settings. 

## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: industry_benchmarks/utils/experimental/plan_rbfe_network_bespoke.py
Comment (IAlibay) [inline industry_benchmarks/utils/experimental/plan_rbfe_network_bespoke.py:256]: Just a rough thought here that we might want to feed a different openff version if the industry partner wants to run benchmarks that match the other bespokefit benchmarks. I'm tempted to say "let's fix it after merge", but I'll leave it up to you to decide if we should wait to merge or not.

**Review by IAlibay — APPROVED** [review]
lgtm thanks!

File: industry_benchmarks/utils/experimental/plan_rbfe_network_bespoke.py
Comment (jthorton) [reply industry_benchmarks/utils/experimental/plan_rbfe_network_bespoke.py:256]: Interesting we could change this to accept any local file or installed force field if they need that functionality, for now, I would say lets do the simple case of local files only and add the other case when needed.

**Pull request merged**
